katago_server/trainings/services/bayesian_elo.py

```diff
@@ -105,12 +104,12 @@ def _add_virtual_draws(self):
                             draw1 = {
                                 "reference_network": network_id,
                                 "opponent_network": other_network_id,
-                                "total_bayesian_virtual_draws": 1.0 / num_other_networks,
+                                "total_bayesian_virtual_draws": 0.01 / num_other_networks,
```

Comment:
Would you mind to explain me this change ?

What is the impact on elo calculation ?

Reply:
The 1.0 strength prior is vastly too strong. So I weakened it. If we truly have no info about a network and no parent, then it should be rated based almost purely on the data, and this tiny weak prior is purely so there is no numerical blowup when trying to rate it at first.